### PR TITLE
Table Graph Integration

### PR DESCRIPTION
**Pull request by nxjiten**
@natabal 
Convert `table graph` from class component to functional component. Please have a look.

### CODE REVIEW

File: GraphsV2/Table/index.js [hunk 1]
```diff
@@ -0,0 +1,1010 @@
...
+                    event: events.REMOVED_COLUMNS
+                }, props.updateScroll);
+            }
+        }
+        initiate(props); 
+        checkFontsize();
+        if (contextMenu) {
+            openContextMenu();
+        }
+        return () => {
```
Comment (natabal) [inline GraphsV2/Table/index.js:844]: so this should executed on unmount. It's very hard to follow. Can you please create a function for this. Call it cleanup?

File: GraphsV2/Table/index.js [hunk 2]
```diff
@@ -0,0 +1,1010 @@
...
+            },
+            selectedRows: {
+                text: "row(s) selected",
+                delete: "Delete",
+                deleteAria: "Delete Selected Rows",
+            },
+        },
+    };
+
+    const muiTableStyle = {
```
Comment (natabal) [inline GraphsV2/Table/index.js:934]: why do we do this here and not useStyle? Please change

File: GraphsV2/Table/tableGraph.spec.js [hunk 3]
```diff
@@ -0,0 +1,797 @@
+import React from 'react';
```
Comment (natabal) [inline GraphsV2/Table/tableGraph.spec.js:1]: you should testcases for context menu and for infobox

**Comment by natabal** [comment]
@nxanil Please review this PR as well. thanks

**Comment by nxjiten** [comment]
@natabal 
I have done the feedback changes and also have done the changes to upgrade the performance.
Please have a look.
